src/riak_kv_qry_compiler.erl

```diff
@@ -2216,4 +2223,12 @@ negate_an_aggregation_function_test() ->
         finalise_aggregate(Select, [3])
       ).
 
+doctored_coverage_context_test() ->
```

Comment:
Is the test asserting the error, it hard to tell from the title. `invalid_coverage_context_checksum_test` might be easier to understand without looking at the entire test.